src/usr/local/www/interfaces.php

```diff
@@ -1695,11 +1698,10 @@ function build_gatewayv6_list() {
 					gettext("Don't forget to adjust the DHCP Server range if needed after applying."));
 }
 
-if ($savemsg) {
```

Comment:
This savemsg code has been sitting unused, because the code for doing "Apply" never set up $savemsg.